tests/cupyx_tests/scipy_tests/fftpack_tests/test_fftpack.py

```diff
@@ -88,6 +88,38 @@ def test_fft2_overwrite(self, xp, scp, dtype):
         return scp.fftpack.fft2(x, shape=self.s, axes=self.axes,
                                 overwrite_x=True)
 
+    @testing.for_complex_dtypes()
+    @testing.numpy_cupy_allclose(rtol=1e-4, atol=1e-7, accept_error=ValueError,
+                                 contiguous_check=False, scipy_name='scp')
+    def test_fft2_plan(self, xp, scp, dtype):
+        x = testing.shaped_random(self.shape, xp, dtype)
+        if isinstance(scp, cupyx.scipy):
```

Comment:
`scp is cupyx.scipy` or `issubclass(scp, cupyx.scipy)`.

Reply:
done